ui/imports/shared/popups/send/panels/HoldingSelector.qml

```diff
@@ -95,16 +92,18 @@ Item {
         property var assetComboBoxModel: SortFilterProxyModel {
             sourceModel: root.assetsModel
             filters: [
-                ExpressionFilter {
-                    expression: {
-                        d.uppercaseSearchText; // Force re-evaluation when searchText changes
```

Comment:
This should still be there

Reply:
its not needed as its passed during the function call and reevaluated because of it